From b093ea65f89ab91db60ec0c698182b184ac1abcd Mon Sep 17 00:00:00 2001
From: jrior001 <jriordan001@gmail.com>
Date: Sat, 6 Oct 2018 00:35:54 -0400
Subject: [PATCH 04/10] move snap_app type definition to public

* this must be pulic to be used outside of here

Change-Id: Ic6f3526cdeeec9f7fafe0cb666b7f5fec7682ea7
---
 common/private/snap_app.te | 2 --
 common/public/snap_app.te  | 1 +
 2 files changed, 1 insertion(+), 2 deletions(-)
 create mode 100644 common/public/snap_app.te

diff --git a/common/private/snap_app.te b/common/private/snap_app.te
index 2dce827..bdd1040 100644
--- a/common/private/snap_app.te
+++ b/common/private/snap_app.te
@@ -1,5 +1,3 @@
-type snap_app, domain, coredomain;
-
 app_domain(snap_app)
 net_domain(snap_app)
 
diff --git a/common/public/snap_app.te b/common/public/snap_app.te
new file mode 100644
index 0000000..5843d47
--- /dev/null
+++ b/common/public/snap_app.te
@@ -0,0 +1 @@
+type snap_app, domain, coredomain;
-- 
2.17.1

